pyxnat/core/derivatives/validator.py

```diff
@@ -0,0 +1,10 @@
+XNAT_RESOURCE_NAMES = ['BBRC_VALIDATOR']
+
+def tests(self, name, key=None):
+    import json
+    j = [e for e in list(self.files(name)) if 'json' in e._urn][0]
```

Comment:
```suggestion
    j = [e for e in list(self.files('{}*.json'.format(name)))][0]
```

Reply:
I find it hard for end-user to provide full matching file name (which is composed validator name and MRSession label)